docs/concepts/Auditing-Packages.md

```diff
@@ -125,6 +125,15 @@ If security vulnerabilities are found and updates are available for the package,
 - Use the NuGet package manager user interface in Visual Studio to update the individual package.
 - Run the `dotnet add package` command with the respective package ID to update to the latest version.
 
+#### Transitive Packages
+
+If a known vulnerability exists in a top-level package's transitive dependencies, you have these options:
+
+- Add the fixed package version as a direct package reference. **Note:** Be sure to remove this reference when a new package version update becomes available.
```

Comment:
We should also tell folks to take care when doing this that they don't change the deployment behavior of their project.  I've seen folks hoist a dependency that was marked with PrivateAssets and ExcludeAssets and forget to add those same markings.

Reply:
Something like?

Note: This does not change the deployment behavior in any way.

Reply:
No, I mean the user needs to *take care* to make sure that they don't do this.  So if they were previously doing:
```xml
<PackageRefence Include="Direct" Version="1.0.0" PrivateAssets="All" ExcludeAssets="Runtime" />
```

They should take care when adding a new reference to give it the same:
```xml
<PackageRefence Include="Direct" Version="1.0.0" PrivateAssets="All" ExcludeAssets="Runtime" />
<!-- Note: Apply the same values for PrivateAssets and ExcludeAssets as Direct --> 
<PackageRefence Include="Indirect" Version="1.1.0" PrivateAssets="All" ExcludeAssets="Runtime"/>
```

It's not automatic here.  It is for CPM - that's a major benefit it gives.

Reply:
ah i see. you're talking about the attribute behavior.

@jeffkl @zivkan should we explicitly mention this?

Reply:
Maybe we say: Take care when adding the dependency to match the [Assets settings](https://learn.microsoft.com/en-us/nuget/consume-packages/package-references-in-project-files#controlling-dependency-assets) of the existing dependency.  When doing so it should behave in the same way as if your dependency itself was updated.  This is not required when using CPM.

Reply:
yep. "whne adding a explicit reference, ensure you maintain your packagereference attributes for the asset behavior" etc

Reply:
I'm confused, or I think we have inconsistent behaviour (which now would be a breaking change to fix).  Today, so coincidence it's on the same day, I was pinged on Discord because someone claims that a package reference with `ExcludeAssets="runtime"` does not exclude that package's dependencies runtime assets, and they had to manually list every transitive package as direct references and add the same `ExcludeAssets="runtime"`.

If CPM's central pinning does flow through include/private/exclude assets, then it sounds like it changes behaviour compared to the same package not being pinned, which feel like a bug to me.

Reply:
Here's a good example of the sort of thing that makes this complicated: https://github.com/dotnet/aspnetcore/pull/57560#issuecomment-2321908053